packages/server/modules/core/tests/generic.spec.js

```diff
@@ -147,17 +156,15 @@ describe('Generic AuthN & AuthZ controller tests', () => {
     })
 
     it('should get the passed in role for server:admins if override enabled', async () => {
-      while (!adminOverrideEnabled()) {
-        process.env.ADMIN_OVERRIDE_ENABLED = 'true'
-      }
+      envHelperMock.enable()
+      envHelperMock.mockFunction('adminOverrideEnabled', () => true)
```

Comment:
Looks good, but FYI you can use `hijackFunction()` instead to mock a function only for a single (or X) invocations so that you don't have to disable it afterwards.